nrf_modem/README.rst

```diff
@@ -2,33 +2,29 @@
 
 Modem library
 #############
-The Modem library is the primary interface for operating the nRF9160 modem, establishing the LTE-M and NB-IoT connections, and receiving the position data (GPS).
-The library abstracts away the details of memory management and synchronization, allowing you to focus on writing the application using standardized API calls.
 
-The library aims to be compatible with the socket standard to simplify writing and porting of existing applications.
-The library supports up to eight networking sockets.
+The Modem library is the primary interface for operating the nRF9160 modem, establishing the LTE-M and NB-IoT connections, and receiving the position data (GPS).
+The library aims to be compatible with the POSIX socket APIs to simplify writing and porting of existing networking applications.
 
-To get started with socket programming, you should check out some introductions and tutorials, such as `Beej's Guide to Network Programming`_.
+To get started with socket programming, you must check out some introductions and tutorials, such as `Beej's Guide to Network Programming`_.
```

Comment:
`must` is wrong here, `should` was okay but `can` is the most correct imo; this is purely a recommendation, there are no requirements whatsoever to look at that specific document.

Reply:
Agree.